### PR TITLE
Add Swoft FrameWork

### PR DESCRIPTION
**Pull request by wujunze**
Modern High performance AOP and Coroutine PHP Framework, base on Swoole

### CODE REVIEW

**Comment by wujunze** [comment]
@ziadoz   Please merge   Thanks

File: README.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Yii2](https://github.com/yiisoft/yii2/) - Another PHP framework.
 * [Zend Framework 2](https://framework.zend.com) - Another framework comprised of individual components (ZF2).
 * [Ice](https://www.iceframework.org/) - Another simple and fast PHP framework delivered as C-extension.
+* [Swoft](https://github.com/swoft-cloud/swoft) - Modern High performance AOP and Coroutine PHP Framework, base on Swoole
```
Comment (dereuromark) [inline README.md:127]: Full stop missing at the end.

Comment (wujunze) [reply README.md]: fixed

**Review by ziadoz — APPROVED** [review]

**Comment by ziadoz** [comment]
The English docs page here doesn't appear to work. http://swoft.io/

**Comment by dereuromark** [comment]
Seems dead.

**Pull request closed without merging**